src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -69,6 +70,49 @@ if (__DEV__) {
     validateInputPropertes(type, props);
     validateUnknownPropertes(type, props);
   };
+
+  var warnForTextDifference = function(serverText: string, clientText: string) {
+    if (didWarnInvalidHydration) {
+      return;
+    }
+    didWarnInvalidHydration = true;
+    warning(
+      false,
+      'Text content did not match. Server: "%s" Client: "%s"',
+      serverText,
+      clientText,
+    );
+  };
```

Comment:
Sort of beside the point, but I'm curious if this warning gets lifted out with uglify. Has there been any consideration to a `warnOnce`?